test/helpers.test.js

```diff
@@ -66,3 +87,22 @@ function testValidationAcceptsValue(validationFn, value) {
 function testValidationRejectsValue(validationFn, value) {
   expect(validationFn(value), value).to.be.a('string');
 }
+
+function testRelationAcceptsValue(modelDefinition, value) {
+  checkRelationName(modelDefinition, value, function(result) {
+    expect(result).to.equal(true);
+  });
+}
+
+function testRelationRejectsValue(modelDefinition, value) {
+  checkRelationName(modelDefinition, value, function(result) {
+    expect(result, value).to.be.a('string');
+  });
+}
+
+function ModelDefinition(propertyList) {
+  this.propertyList = propertyList;
+  this.properties = function(callback) {
+    callback(null, this.propertyList);
+  };
```

Comment:
We usually don't mock LoopBack models. However, since setting up temporary workspace models is non-trivial, I think it's ok to make an exception here.